### PR TITLE
Log more details when codebuild job fails

### PR DESCRIPTION
**Pull request by fouadchamoun**
this allows for more information on the exact reason for failure on the AWS Codebuild side.

Example of previous problematic behavior: a timeout on codebuild is not reported and the user (on gitlab) does not know why the job suddenly stopped.

### CODE REVIEW

**Comment by fouadchamoun** [comment]
tested on this pipeline (private repo): https://gitlab.greenflex.com/tools/qlik-backup-management/-/jobs/488235

File: tests/print_events.py [hunk 1]
```diff
@@ -1,13 +0,0 @@
-import sys
```
Comment (ebarault) [inline tests/print_events.py:1]: why removing this ?

Comment (fouadchamoun) [reply tests/print_events.py:1]: I believe this script was copied from the scripts dir to the tests dir, in order to be able to invoke it in a test env. 
But I've refactored the code to handle this more gracefully and not have to duplicate the scripts for testing purposes. See https://github.com/ebarault/anyservice-awscode-docker/pull/8/files#diff-3d2084d2983188c5267b5b8203dd9941523c1ded70a99bb53fab8db6926b65eeR186

Comment (ebarault) [reply tests/print_events.py:1]: ok

**Pull request merged**